.gitignore

```diff
@@ -1,62 +1,66 @@
-# Python
-*.pyc
-__pycache__/
-venv/
-*.egg-info/
-dist/
-build/
-*.egg
-
-# C++
-*.o
-*.out
-*.exe
-*.dll
-*.so
-*.dylib
-
-# Compiled files
-*.class
-*.jar
-
-# IDE and editor files
-.vscode/
-.idea/
-*.suo
-*.ntvs*
-*.njsproj
-*.sln
-*.swp
-*.swo
-*.bak
-*.tmp
-*.tmp.*
-*.log
-*.cache
-*.sublime-workspace
-
-# Model
-*.pt
-
-# Dependency directories
-node_modules/
-bower_components/
-
-# Images directories
-classification_output/
-detection_output/
-url_imgs/
-debug/
-wadas_motion_detection/
-demo_output/
-
-# Images
-#*.png
-#*.jpg
-#*.jpeg
-*.gif
-#*.ico
-*.svg
-
-
-cache
+# Python
+*.pyc
+__pycache__/
+venv/
+*.egg-info/
+dist/
+build/
+*.egg
+
+# C++
+*.o
+*.out
+*.exe
+*.dll
+*.so
+*.dylib
+
+# Compiled files
+*.class
+*.jar
+
+# IDE and editor files
+.vscode/
+.idea/
+*.suo
+*.ntvs*
+*.njsproj
+*.sln
+*.swp
+*.swo
+*.bak
+*.tmp
+*.tmp.*
+*.log
+*.cache
+*.sublime-workspace
+
+# Model
+*.pt
+*.onnx
+*.bin
+*.xml
+
+# Dependency directories
+node_modules/
+bower_components/
+
+# Images directories
+classification_output/
+detection_output/
+url_imgs/
+debug/
+wadas_motion_detection/
+demo_output/
+

```

Comment:
Pristine (or add, i don't see it in previous version but was supposed to be there) the "log" folder, where all logs files are stored. E.g., WADAS main log, FTP server log, etc..